.github/workflows/experimental_build.yml

```diff
@@ -50,8 +50,8 @@ jobs:
           tools/run_tests.sh all
 
   build_job_gcc54:
-    runs-on: ubuntu-16.04
-    name: Build on Ubuntu 16.04
+    runs-on: ubuntu-18.04
```

Comment:
I am fine with upgrading to 18.04 but why are we doing this?

Reply:
Because GitHub does not provide Ubuntu 16 runners anymore